td.desktop/core/services/joint.shapes.tm.js

```diff
@@ -258,7 +258,7 @@ joint.shapes.tm.Process = joint.shapes.tm.toolElement.extend({
 
 //define process element properties
 
-joint.shapes.tm.utils.defineProperties(joint.shapes.tm.Process.prototype, ['privilegeLevel']);
+joint.shapes.tm.utils.defineProperties(joint.shapes.tm.Process.prototype, ['privilegeLevel', 'handlesCardPayment', 'isWebApplication', 'handlesGoodsOrServices']);
```

Comment:
Are these all new properties being added to entities within the diagram? Should I include these and "storesInventory" in 2.0 as well?

Reply:
@lreading yes, good point, we needed to add some more context to the Process element and another to the Store element

If you could add these to version 2.0 also that would be great. I am vaguely worried that we might change our mind on some of these properties though, especially on the naming and the json entry